config/navigations/services_navigation.rb

```diff
@@ -286,11 +286,11 @@
                  'Services',
                  nil,
                  html: { class: 'fancy-nav-header', 'data-icon': 'service-icon' },
-                 if: -> { services.available?(:email_service) } do |services_nav|
+                 if: -> { plugin_available?(:email_service) } do |services_nav|
```

Comment:
why did you changed the `services.available?` check? I would check if the service `cronus` is available and as a fall back check if the plugin exists. See here:
https://github.com/sapcc/elektra/blob/81a55a49aba95b8c6fc85e03c021f44fc5c48b02/config/navigations/services_navigation.rb#L64
Afterwards you should define the `available?` method in your service layer.
https://github.com/sapcc/elektra/blob/81a55a49aba95b8c6fc85e03c021f44fc5c48b02/plugins/compute/app/services/service_layer/compute_service.rb#L21

Reply:
done.